### DISCUSSION TITLE
[ FF8 ] Crashing on first enemy encounter

### DISCUSSION THREAD

**Discussion by Geekgamerinc** [julianxhokaxhiu/FFNx#752] [General]
**PLEASE NOTE:** If you come here to report a crash while you were using 7th Heaven, please do on their relative support channels. NO SUPPORT will be provided, and your issue will be close immediately.

**Describe the bug**
When exiting Balamb Garden, heading to the Fire Cavern, the game crashes reliably on the first random encounter in the overworld.

**To Reproduce**

1. Go through intro, go out to overworld
2. Walk around to encounter first creature for battle.
3. Enter battle
4. Game stutters and crashes

**Expected behavior**
Exactly as explained above.

**Screenshots**

**GPU (please complete the following information):**

- Brand Nvidia GeForce RTX 4060 Ti
- Driver Version 566.03

**Additional context**
I installed FFNx, as well as Ragnarok 1.2.3 for 2013 version. I'm playing the original 2013 release, NOT THE REMASTERED.
FFNx settings are
1920x1080 16:9
16:9 Stretched
16x Anti-Aliasing
[FFNx.log](https://github.com/user-attachments/files/17724492/FFNx.log)

**Comment by julianxhokaxhiu** [comment]
Can you please link where you found the "Ragnarokv" mod? I don't recognize that one. Thank you in advance

**Reply by Geekgamerinc** [reply]
Apologies, I fat fingered that. It's Ragnarok 1.2.3 from Qhimm user Calisto:
https://forums.qhimm.com/index.php?topic=18404.msg258999#msg258999

**Reply by julianxhokaxhiu** [reply]
Since you're using a mod that touches the game data and binary code in memory, you better reach the mod author to ask to check that his mod works with FFNx. We have not tested this combination and we don't have the resources to test every possible mod combination, sorry. Feel free to test the same scenario again only with FFNx and if there is an issue feel free to report, we test only the vanilla game with FFNx on top. Hope this helps, cheers!

**Reply by Geekgamerinc** [reply]
Oddly enough, I did a complete uninstall, and re-install of ff8, then installed FFNx and it works now... very strange. Ragnarok is still very much present too. But thank you for the help, I appreciate it!

**Reply by julianxhokaxhiu** [reply]
Glad to hear that, maybe it was just some file corruption during download. Hope you enjoy it :) Cheers!
